skills/docx/ooxml/scripts/validation/base.py

```
@@ -0,0 +1,951 @@
+            with zipfile.ZipFile(self.original_file, "r") as zip_ref:
+                zip_ref.extractall(temp_path)
+
+            # Find corresponding file in original
+            original_xml_file = temp_path / relative_path
+
+            if not original_xml_file.exists():
+                # File didn't exist in original, so no original errors
+                return set()
+
+            # Validate the specific file in original
+            is_valid, errors = self._validate_single_file_xsd(
```

Ruff (0.14.3)

Unpacked variable `is_valid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)